### ISSUE TITLE
NeedBraces property allowEmptyLoopBody not giving error on empty loop body.

### ISSUE THREAD

**Issue by Vyom-Yadav** [checkstyle/checkstyle#9123]
https://checkstyle.org/config_blocks.html#NeedBraces
Using checkstyle 8.38 with config-
```
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
        "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
        "http:///www.puppycrawl.com/dtds/configuration_1_3.dtd">
<module name="Checker">
    <module name="TreeWalker">
        <module name="NeedBraces">
            <property name="allowEmptyLoopBody" value="FALSE"/>
            <property name="tokens" value="LITERAL_FOR"/>
        </module>
    </module>
</module>
```
Code I used
```
public class Test {
    void check() {
        for (int j = 0; j < 10; j++){

        }

    }
}
```
Doesn't give an error for an empty loop body.
```
D:\workspace\configCheckstyle\src\cC>java -jar E:/Downloads/checkstyle-8.38-all.jar -c config.xml Test.java
Starting audit...
Audit done.
```

**Comment by strkkk** [comment]
I do not understand the problem. Check demands curly braces for loop, and loop has curly braces, looks like ok.

**Comment by Vyom-Yadav** [comment]
The property `allowEmptyLoopBody` allows loops with empty bodies but here when its value Is false, it should not allow loop with empty body, but here it's allowing.

**Comment by Vyom-Yadav** [comment]
Sorry my bad, I was expecting the check to do something else.
